### PR TITLE
Add Cursor's Recent Projects, Open With Cursor, Show Installed Extensions & Open New Window commands

### PR DESCRIPTION
**Pull request by degouville**
## Description

Now that Cursor has evolved into a standalone product beyond "another VSCode fork," we increasingly use it as an AI code editor with its unique workflow, particularly the Composer feature.

After numerous requests from the community and my personal experience switching between Cursor and VSCode, I believe this fork of the VSCode extension for Cursor will add valuable handy possibilities.
Now you can:
- Set distinct **Hotlinks** for VSCode and Cursor
- Maintain separate **Recent Projects** lists
- See the authentic Cursor logo for commands
- Quickly open projects in either VSCode or Cursor

## Screencast
Screenshot 2024-09-12 at 20 03 33

Screenshot 2024-09-12 at 20 04 06

Screenshot 2024-09-12 at 20 04 29

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by degouville** [comment]
up @thomaspaulmann @pernielsentikaer

**Comment by pernielsentikaer** [comment]
I think we can allow it to live as a standalone extension now that Cursor has received so much attention. However, you'll need to clear up all contributors and add it to its own folder instead - does that make sense?

As we discussed, it's basically a fork of the VSCode extension, so there might be things that need to be fixed in multiple places

**Comment by degouville** [comment]
Hi @pernielsentikaer,

Thanks for your answer.

> However, you'll need to clear up all contributors

Sure! I wanted to salute of VSCode's contribs

> add it to its own folder instead

I'm not sure I understand that one. Do you have any advice on how to get those views outside the `.../cursor/` folder ?
Or is it about the contributors list ? Can you please reformulate ?

> so there might be things that need to be fixed in multiple places

I took some time to refactor the parts that were necessary for the "Build Selection" and remove unneeded features for direct Cursor features. I think we are good on that but please let me know if you have a second guess.

**Comment by pernielsentikaer** [comment]
So /cursor/ is the current extension that shows the documentation. Instead of adding it to this extension, I think we should create a new extension so users who have installed the documentation extension don't get confused

Does that make sense?

**Comment by degouville** [comment]
Ok I get it.

I'm keeping the 1st documentation command in `/cursor/`

Then put the new commands in `/cursor-recent-projects/`

**Comment by degouville** [comment]
Here we go @pernielsentikaer ✅

File: extensions/cursor/package.json [hunk 1]
```diff
@@ -1,37 +1,128 @@
...
+		"Developer Tools",
+		"Documentation",
+		"Productivity"
+	],
+	"keywords": ["cursor", "vscode", "claude", "sonnet", "opus", "o1", "gpt4o"],
+	"license": "MIT",
+	"commands": [
+		{
+			"name": "index",
+			"title": "Search Recent Projects",
```
Comment (pernielsentikaer) [inline extensions/cursor/package.json:19]: Is this correct name?

Comment (pernielsentikaer) [reply extensions/cursor/package.json:19]: Seems like this file needs some more love

**Comment by degouville** [comment]
All done @pernielsentikaer  ✅

**Comment by degouville** [comment]
Updated @pernielsentikaer

**Comment by degouville** [comment]
Done ✅

**Comment by pernielsentikaer** [comment]
The command `Open New Window` keeps getting me this error 
Raycast22092024-I3PTf0iA png@2x

Can you look into why, do you need more from me?

**Comment by degouville** [comment]
I can't reproduce this bug @pernielsentikaer 

https://github.com/user-attachments/assets/745d7f27-f836-4c73-8949-914d5444f5cf

Can you please please share your result of this command please ?

```bash
which Cursor
/usr/local/bin/Cursor
```

**Comment by degouville** [comment]
Do you get the command "Open with Cursor" working ??

**Comment by degouville** [comment]
I can't see any issue here.

Can you please try the https://www.raycast.com/thomas/visual-studio-code command `Open new Window` with Cursor as selected build to make sure the issue isn't on you install ?

**Comment by pernielsentikaer** [comment]
Sorry about that! I found the issue; it was my fault. I had to test another extension and had accessibility turned off and completely forgot about it 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**